frontend/tasks/scripts.js

```diff
@@ -58,36 +64,66 @@ gulp.task('scripts-generate-static-html', () => {
     debug: config.IS_DEBUG,
     fullPaths: config.IS_DEBUG,
     transform: [babelify],
-    standalone: 'generate-static-html'
-  }).external(vendorModules));
+    standalone: 'generate-static-html',
+    bundleExternal: false
+  }).external(vendorModules), config.DEST_PATH);
 });
 
-gulp.task('scripts-app-main', () => {
+gulp.task('scripts-app', () => {
   return commonBrowserify('app.js', browserify({
-    entries: [config.SRC_PATH + 'app/index.js'],
+    entries: [config.SRC_PATH + 'app.js'],
```

Comment:
Just realized this moved... I think it's because `bootstrap.js` came & went. This should probably be moved back to `frontend/src/app/index.js` rather than `frontend/src/app.js`